### PR TITLE
Update ADOPTERS.md to include UiPath

### PR DESCRIPTION
**Pull request by mjnovice**
### Description of your changes

Fixes # 

I have: 

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added or updated unit tests.
- [x] Added or updated e2e tests.
- [x] Linked a PR or a [docs tracking issue] to [document this change].
- [x] Added `backport release-x.y` labels to auto-backport this PR.

Need help with this checklist? See the [cheat sheet].

[contribution process]: https://github.com/crossplane/crossplane/tree/master/contributing
[docs tracking issue]: https://github.com/crossplane/docs/issues/new
[document this change]: https://docs.crossplane.io/contribute/contribute
[cheat sheet]: https://github.com/crossplane/crossplane/tree/master/contributing#checklist-cheat-sheet

### CODE REVIEW

**Review by negz — APPROVED** [review]
Thank you!

File: ADOPTERS.md [hunk 1]
```diff
@@ -81,4 +81,5 @@ This list is sorted in the order that organizations were added to it.
...
-| [AlphaSense](https://www.alpha-sense.com/) | @abhihendre | Engineering teams at [AlphaSense](https://www.alpha-sense.com/) leverage Crossplane APIs, abstracted by a set of Helm charts and Compositions curated by our Platform Teams, to seamlessly provision cloud services across three major clouds, including our production environment.|
\ No newline at end of file
+| [AlphaSense](https://www.alpha-sense.com/) | @abhihendre | Engineering teams at [AlphaSense](https://www.alpha-sense.com/) leverage Crossplane APIs, abstracted by a set of Helm charts and Compositions curated by our Platform Teams, to seamlessly provision cloud services across three major clouds, including our production environment.|
+| [UiPath](https://www.uipath.com/) | [@mjnovice](https://github.com/mjnovice) | Control plane for infrastructure management which powers [AutomationSuite](https://docs.uipath.com/automation-suite/automation-suite/2023.10/installation-guide-eks-aks/automation-suite-on-eksaks-overview) |
```
Comment (jbw976) [inline ADOPTERS.md:85]: @mjnovice is this also in production environments at UiPath? If so, could you update the text to specifically include something like "in production environments"? That would also helpful to know!

if not, no worries, we can merge as is 😉

**Review by jbw976 — APPROVED** [review]

File: ADOPTERS.md (hunk 1 above)
Comment (mjnovice) [reply ADOPTERS.md:85]: Hey Jared. We have an experimental product which might end up in production. Will update once that happens.

Comment (jbw976) [reply ADOPTERS.md:85]: sounds great, looking forward to an update PR when you get there! 😉

**Pull request merged**